we-dance/platform
# Feat/69

Replacing the textarea in the description field of event editor, with the editor js for giving a UI functionality rather than markup.

### How to test it
Go to the url:
You should see the editor js in action
I don't know why but it's rendering the editor js part twice.
Closes #69
